curriculum/js1/3.js

```diff
@@ -5,10 +5,11 @@
  * @returns {string}
  */
 
-const solution = (a, b) => {
-  return 0;
-};
-
-module.exports = {
-  solution,
+const solution = (x, i = 0, newStr = "") => {
+  if(i === x){
+     return newStr;
+  };
+  newStr = newStr + "hello";
+  return solution(x, i + 1, newStr) 
```

Comment:
there's no point creating a new variable. just pass the new data in!
`return solution(x, x+1, newStr + 'hello');`

Reply:
Are you using the same x for "x" and "x+1" ?

Reply:
yeah 'x' will be the same.... which reminds me, you don't even need `i`. Just decrease x until it becomes 0!